### PR TITLE
Container runtimes

### PR DESCRIPTION
**Pull request by nimrodoron**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

### CODE REVIEW

File: charts/seed-bootstrap/templates/extensions/crd-containerruntime.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+      served: true
+      storage: true
+  version: v1alpha1
+  scope: Namespaced
+  names:
+    plural: containerruntimes
+    singular: containerruntime
+    kind: ContainerRuntime
+    shortNames:
+      - cre
```
Comment (liorokman) [inline charts/seed-bootstrap/templates/extensions/crd-containerruntime.yaml]: Is this really needed? `ContainerRuntime` is esoteric enough that there's no point in creating a short name for it.

**Review by liorokman — APPROVED** [review]
This just adds the new extension type, right?

**Comment by nimrodoron** [comment]
Gardener leftovers:
- Validation webhook
- Containerd unit service (log rotate and monitor)

File: charts/seed-bootstrap/templates/extensions/crd-containerruntime.yaml (hunk 1 above)
Comment (nimrodoron) [reply charts/seed-bootstrap/templates/extensions/crd-containerruntime.yaml]: Removed

**Pull request closed without merging**